plugins/kotlin/j2k/k1.new/src/org/jetbrains/kotlin/nj2k/conversions/RemoveWrongExtraModifiersConversion.kt

```diff
@@ -6,14 +6,23 @@ import org.jetbrains.kotlin.nj2k.NewJ2kConverterContext
 import org.jetbrains.kotlin.nj2k.RecursiveConversion
 import org.jetbrains.kotlin.nj2k.tree.*
 
-internal class RemoveWrongExtraModifiersForSingleFunctionsConversion(context: NewJ2kConverterContext) : RecursiveConversion(context) {
+internal class RemoveWrongExtraModifiersConversion(context: NewJ2kConverterContext) : RecursiveConversion(context) {
     override fun applyToElement(element: JKTreeElement): JKTreeElement {
         if (element !is JKOtherModifiersOwner) return recurse(element)
         if (element.parentOfType<JKClass>() == null) {
+            // single function --> remove STATIC
```

Comment:
Is it really applicable to functions only? If yes, it's better to add an explicit check in the code.